security/hashcat/Portfile

```diff
@@ -25,6 +16,15 @@ long_description        hashcat is the world's fastest and most advanced \
 
 homepage                https://hashcat.net/hashcat/
 
+categories              security
+license                 MIT
+platforms               darwin
+maintainers             @gaming-hacker openmaintainer
+
+checksums               rmd160  c53d308a1b89e1d48ef26d2f9bf1146e9368d5bf \
+                        sha256  92540830c0078254a4b90bdcd6a329add2da85dd85002725d91bd6948a2486ee \
+                        size    5365734
+
 use_configure           no
 
 build.env-append        PREFIX=${prefix}
```

Comment:
the `use_configure` and the `build.env-append`/`destroot.env-append` can be removed  when including the PG

Reply:
Yes, adding that portgroup allowed me to remove both just as you stated.